tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py35, py36, py37, py38, py39, python3.10, pre-commit
+    py35, py36, py37, py38, py39, py3.10, pre-commit
```

Comment:
it should be just `py310`

Reply:
Gnah. I stared at the relevant tox change and got it completely wrong ... le sigh ... Thanks. 

It's broken anyways, so ...